docs/fee-policies.md

```diff
@@ -0,0 +1,96 @@
+# Fee policies
+
+Fee policies provides special fees to be applied for users from specific `Legal Entity`. Usually the fee policy will apply lesser fee amount over the bridge transaction.
+In order to enable fee policies to the bridge - submit _Topic_ and with this _Topic Id_ in `config/bridge.yml` (See [Configuration](configuration.md), field `bridge.config_topic_id`). 
```

Comment:
I'm not sure what is meant by this. Probably something along the lines of:
`submit a topic message with the config to the topic id from the config.`